modules/openstack/limits.go

```diff
@@ -151,3 +161,73 @@ func (o *OpenStack) CreateRegisteredLimit(
 
 	return limitID, nil
 }
+
+// DeleteRegisteredLimit - delete limit from keystone
+func (o *OpenStack) DeleteRegisteredLimit(
+	log logr.Logger,
+	registeredLimitID string,
+) error {
+    log.Info(fmt.Sprintf("Deleting registered limit %s", registeredLimitID))
+	err := registeredlimits.Delete(o.osclient, registeredLimitID).ExtractErr()
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+//GetRegisteredLimit - Get existing registered limit by ID
+func (o *OpenStack) GetRegisteredLimit(
+	log logr.Logger,
+	registeredLimitID string,
+) (RegisteredLimit, error) {
+    log.Info(fmt.Sprintf("Fetching registered limit %s", registeredLimitID))
+	registered_limit, err := registeredlimits.Get(o.osclient, registeredLimitID).Extract()
+    if err != nil {
+        return _, err
```

Comment:
`Error: vet: ./limits.go:186:16: cannot use _ as value or type`

Reply:
Should we change `RegisteredLimit` into something like `*registeredlimits.RegisteredLimit` and return it from the function? Just trying to figure out how to properly change this function.
We could have something like:

```
// GetRegisteredLimit - Get existing registered limit by ID
func (o *OpenStack) GetRegisteredLimit(
	log logr.Logger,
	registeredLimitID string,
) (*registeredlimits.RegisteredLimit, error) {
	log.Info(fmt.Sprintf("Fetching registered limit %s", registeredLimitID))
	registered_limit, err := registeredlimits.Get(o.osclient, registeredLimitID).Extract()
	if err != nil {
		return nil, err
	}
	return registered_limit, nil
}
```
but I have to try the whole flow and double check it works as expected.
Anyway, no need to use "_" in a return statement, we can return "nil" on that field.

Reply:
Same comment applies to the other functions